react/src/components/lablupTalkativotUI/LLMChatCard.tsx

```diff
@@ -157,6 +163,19 @@ const LLMChatCard: React.FC<LLMChatCardProps> = ({
   }, [submitKey]);
 
   const items: MenuProps['items'] = [
+    {
+      key: 'compare',
+      label: t('chatui.CompareWithOtherModels'),
+      icon: <Scale />,
+      onClick: () => {
+        webuiNavigate(
+          `/serving?tab=chatting&endpointId=${endpointId}&modelId=${modelId}`,
+        );
+      },
```

Comment:
`webuiNavigate` has `params` option. With this option, we don't worry about escaping
```
        webuiNavigate('/usersettings', {
          params: {
            tab: 'general',
          },
        });
```

Reply:
It doesn't work ;_;